snapshots/src/clob-liq.ts

```diff
@@ -0,0 +1,108 @@
+import { MapOfCount } from "./interfaces";
+import "cross-fetch/polyfill"; // polyfill vs ponyfill
+import { POLY_INTL_ID } from "./constants";
+
+export interface rewardsInt {
+    epoch: number;
+    market: string;
+    maker_address: string;
+    qfinal: string;
+}
+
+export interface makerScore {
+    maker_address: string;
+    qfinal: string;
+}
+
+export const getMarketsIncludedInEpoch = async (
+    clobUrl: string,
+    epoch: number,
+): Promise<string[]> => {
+    let marketsList: string[] = [];
+    await fetch(
+        `${clobUrl}/markets-included-in-epoch?epoch=${epoch.toString()}`,
+        {
+            method: "GET",
+            headers: {
+                POLY_INTL_ID: POLY_INTL_ID,
```

Comment:
random question, do I need to add this ID for anything on the front end?

Reply:
Yeah we will need to chat about this.